src/styles/global.css

```
@@ -0,0 +1,1023 @@
+    --intro-blur: 0px;
+    --folio-left-x: 0%;
+    --folio-right-x: 0%;
+    --folio-opacity: 1;
+    --panel-a-opacity: 0;
+    --panel-a-y: 28px;
+    --panel-b-opacity: 0;
+    --panel-b-y: 28px;
+    --catalog-opacity: 0;
+    --catalog-y: 72px;
+    --controls-opacity: 0;
+    position: sticky;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)